packages/salesforcedx-vscode-replay-debugger/src/states/frameStateUtil.ts

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {
+  EVENT_CODE_UNIT_FINISHED,
+  EVENT_CODE_UNIT_STARTED,
+  EVENT_CONSTRUCTOR_ENTRY,
+  EVENT_CONSTRUCTOR_EXIT,
+  EVENT_VF_APEX_CALL_END,
+  EVENT_VF_APEX_CALL_START,
+  SFDC_TRIGGER
+} from '../constants';
+
+export class FrameStateUtil {
```

Comment:
Can you share some before and after screenshots to help me visualize what this util is doing?

Reply:
No. I don't see the need to revert my changes locally to get before screen shots. I spent time explicitly commenting each and every case, giving examples of what the log lines are and what the framename would become. This is further shown by the unit tests that were added.

Reply:
Unsure why you have to revert. You were already researching the nuances of frame names and gathering samples, so you'd already have the before-screenshots.

It's also good practice to sync with the product side to show what you've implemented is what they had in mind. That conversation usually involves screenshots, so I thought you may have the after-screenshots also.